src/Products/Spacecrafts/HybridSpaceCrafts/HybridSpaceCrafts.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyperdrive\Products\Spacecrafts\HybridSpaceCrafts;
+
+use Hyperdrive\Products\SpacecraftElements\Engines\Engine;
+use Hyperdrive\Products\SpacecraftElements\FuelTanks\FuelTank;
+use Hyperdrive\Products\SpacecraftElements\SpaceCraftElement;
+use Hyperdrive\Products\SpacecraftElements\Weapons\Weapon;
+use Hyperdrive\Products\Spacecrafts\Cargoable;
+use Hyperdrive\Products\Spacecrafts\Spacecraft;
+use Hyperdrive\Products\Spacecrafts\Weaponable;
+
+abstract class HybridSpaceCrafts extends Spacecraft
```

Comment:
Z tego co widzę, to ta klasa reprezentuje jeden statek kosmiczny więc lepiej nazwać ją w liczbie pojedynczej:
```suggestion
abstract class HybridSpaceCraft extends Spacecraft
```